keyboards/handwired/hnah40rgb/hnah40rgb.c

```diff
@@ -0,0 +1,46 @@
+#include "hnah40rgb.h"
+#include "config.h"
+#if defined (ansi)
```

Comment:
This entire block should be wrapped by an ifdef for `RGB_MATRIX_ENABLE`, so that if the feature is disabled, it will still compile.

I know that it isn't likely .... but it is still a good idea.